docs/admin/authorization.md

```diff
@@ -33,30 +33,36 @@ If multiple modes are provided the set is unioned, and only a single authorizer
 
 will always allow.
 
-## ABAC Mode
-
-### Request Attributes
+## Request Attributes
 
 A request has the following attributes that can be considered for authorization:
 
   - user (the user-string which a user was authenticated as).
   - group (the list of group names the authenticated user is a member of).
+  - "extra" (a map of arbitrary string keys to string values, provided by the authentication layer)
   - whether the request is for an API resource.
   - the request path.
-    - allows authorizing access to miscellaneous endpoints like `/api` or
-`/healthz` (see [kubectl](#kubectl)).
+    - allows authorizing access to miscellaneous non-resource endpoints like `/api` or `/healthz` (see [kubectl](#kubectl)).
```

Comment:
Either request path is used, or verb/resource/subresource/group are used, but not both (IIRC).